### ISSUE TITLE
[BITV] 9.4.1.2/5.4 - The "Activity settings" button is an accordion button that should be implemented as such with the aria-expanded attribute.

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/activity#1110]
See https://www.w3.org/WAI/ARIA/apg/patterns/accordion/examples/accordion/ for further details. (1)

- [ ] Fix
- [ ] Backport

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-bfbf07e5c3-v5-n4

</details>
